search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -71,10 +71,11 @@ function toggleDisplay(id) {
 }
 
 /** 
- * Displays the top results in a country for current search trend on modal. 
+ * Displays the top results in a country for current search trend on modal
+ * and show positive/negative words depending on the sign of the sentiment score.  
```

Comment:
Check line wrapping.
```suggestion
 * and show positive/negative words if they match the sign of the sentiment score.  
```